packages/test/test-gc-sweep-tests/src/dataObjectWithCounter.ts

```diff
@@ -40,6 +41,19 @@ export class DataObjectWithCounter extends DataObject {
     public stop() {
         this.isRunning = false;
     }
+
+    public start() {
+        this.isRunning = true;
+        this.sendOps().catch((error) => { console.log(error); });
```

Comment:
Consider adding `.then(() => { assert(!this.isRunning); })`

Reply:
I guess you'd have to override `dispose` to set `this.isRunning = false`.  Might not be a bad idea anyway.

Reply:
Trying not to overcomplicate this. If it's not necessary, I'd prefer not to have it in. We can always revisit later

Reply:
I mean it's not extra functionality, it's extra validation that your state machine doesn't have holes.  Up to you

Reply:
I'll leave this for later